src/routes/index.js

```diff
@@ -17,6 +18,7 @@ const publicRoutes = [
     { path: '/detail', page: DetailPage, layout: MainLayout },
     { path: '/about-us', page: 'about us here', layout: MainLayout },
     { path: '/predict', page: DetectionPage, layout: MainLayout }, // cai nay de test, sau nay chuyen ve privateRouter
+    { path: '/bloglist', page: BlogListPage, layout: MainLayout }, // cai nay de test, sau nay chuyen ve privateRouter
```

Comment:
we have a route `/blog` before. You can use it instead of this one.

Reply:
anything wrong right? the blog path is belong to blogPage, so its still good if we have 2 component in 1 path